log/rotation/file.go

```diff
@@ -83,11 +83,14 @@ func (f *File) write(p []byte) (int, error) {
 	}
 	if f.f == nil {
 		var err error
-		f.f, err = os.OpenFile(f.base, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0666)
+		f.f, err = os.OpenFile(f.base, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644) // #nosec
+		if err != nil {
+			return 0, err
+		}
+		f.w, err = f.f.Seek(0, os.SEEK_END)
```

Comment:
Switched the `2` literal to the `os.SEEK_END` constant here, FYI, in addition to handling the error as opposed to adding a `#nosec` (I believe an `EBADF`-style error could possibly occur here)